README.md

```diff
@@ -30,6 +31,61 @@ If you aren't using a debugger (or want to use other debugging configurations),
   </ol>
 </details>
 
+<!-- Quickstart -->
+<details open="open">
+  <summary><h2 style="display: inline-block" id="quickstart">Quickstart</h2></summary>
+
+This quickstart assumes that you've got a [Raspberry Pi
+Pico](https://www.raspberrypi.com/products/raspberry-pi-pico/) (the first-generation
+version containing the RP2040 MCU) as well as a [Raspberry Pi Debug
+Probe](https://www.raspberrypi.com/products/debug-probe/) and will flash the Pico with
+[probe-rs](https://probe.rs/).
```

Comment:
Well, you could generate it using elf2uf2-rs and only have an RP2040 board and a usbcable :)

Reply:
SGTM to switch to the lowest-barrier-to-entry hardware setup if you think new users would prefer that. I think I assumed Pico / Debug Probe / probe-rs because my reading of the README suggested that that is the most common and well-supported setup

LMK which combo you want the quickstart to cover. I can add a note along the lines of "another common combo is X; see this part of the README for more details"

Reply:
lowest barrier to entry is definitely the elf2uf2 route. But it is also pretty limited. E.G. getting logs out of it would require to wire a uart of setup a usb-cdc device.

So the most convenient route to actually develop & "debug" is to have an rtt channel with probe-rs & a debug probe.
This route also unlocks the ability to step-debug if that is really needed (which I am yet to need with rust).